be/src/common/config.h

```diff
@@ -30,6 +30,9 @@ namespace config {
     // port for brpc
     CONF_Int32(brpc_port, "8060");
 
+    //the num of bthreads for brpc, the default value is set to -1, which means the num of bthreads is #cpu-cores
```

Comment:
```suggestion
    //the number of bthreads for brpc, the default value is set to -1, which means the number of bthreads is #cpu-cores
```Keep consistent with other variables

Reply:
fix